cms/search/publishers.py

```diff
@@ -0,0 +1,202 @@
+import json
+import logging
+from abc import ABC, abstractmethod
+
+from django.conf import settings
+from kafka import KafkaProducer
+
+logger = logging.getLogger(__name__)
+
+
+class BasePublisher(ABC):
```

Comment:
this needs a link to the search spec at the very least.
I suggest adding a docs page about the integration with the relevant detail

Reply:
Will get to this at the end =)

Reply:
Added relevant links for now!